packages/api/endpoints/collections.js

```diff
@@ -162,22 +219,17 @@ async function put({ params: { name, version }, body }, res) {
  * @returns {Promise<Object>} the promise of express response object
  */
 async function del(req, res) {
-  const { name, version } = req.params;
+  const {
+    collectionsModel = new models.Collection(),
+    dbClient = await db.connection.knex(),
+  } = req.testContext || {};
```

Comment:
Even after looking at the tests, I'm not sure I can tell what purpose this dependency injection is serving. But I'm also a bit wary of having this code in here just for testing purposes